lib/policy_machine_storage_adapters/template.rb

```diff
@@ -173,5 +173,11 @@ def scoped_privileges(user_or_attribute, object_or_attribute)
 
     end
 
+    # Optimized version of PolicyMachine#accessible_objects
+    # Return all objects the user has the given operation on
+    # Optional: only add this method if you can do it better than policy_machine.rb
+    def accessible_objects(user_or_attribute, operation, options = {})
```

Comment:
Not related to outstanding pr, but why did we do this instead of inheritance + `NotImplemented` errors again?